File: src/java/MySQL_block.java
import java.sql.*;
import java.util.Locale;

public class MySQL_block {
  public static void main(String[] args) {
    Connection conn;
    Statement stmt;
    String query = "";
    int i;
    double x, prob;
    Locale.setDefault(new Locale("en", "US"));
    try {
      conn = DriverManager.getConnection("jdbc:mysql://localhost/gaussian?allowMultiQueries=true", "root", "root123");
      stmt = conn.createStatement();
      stmt.executeUpdate("DROP TABLE IF EXISTS java_block; CREATE TABLE java_block(id INTEGER AUTO_INCREMENT " +
      "PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)");
      for (i = -500; i <= 500; i++) {
        x = (double) i / 100.0;
        prob = Gaussian.gaussianCDF(0, 1, x);
        query += String.format("INSERT INTO java_block(z_score, cumulative_distribution) VALUES (%.2f, %f);", x, prob);
      }
      stmt.executeUpdate(query);
    } catch (Exception e) {
      System.err.println(e.getMessage());
    }
  }
}

File: src/php/mysql_block.php
<?php

include "gaussian.php";

try {
  $conn = new mysqli("localhost", "root", "root123", "gaussian", 3306);
  $query = "";
  $conn->multi_query("DROP TABLE IF EXISTS php_block; CREATE TABLE php_block(id INTEGER AUTO_INCREMENT
  PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)");
  for ($i = -500; $i <= 500; $i++) {
    $x = $i / 100;
    $prob = gaussianCDF(0, 1, $x);
    $query .= sprintf("INSERT INTO php_block(z_score, cumulative_distribution) VALUES (%.2f, %f);", $x, $prob);
  }
  $conn->next_result();
  $conn->multi_query($query);
  $conn->close();
} catch (\Throwable $th) {
  echo $th->getMessage();
}
